RadarSDK/RadarGeofence.m

```diff
@@ -228,7 +228,7 @@ - (NSDictionary *)dictionaryValue {
         RadarCircleGeometry *circleGeometry = (RadarCircleGeometry *)self.geometry;
         [dict setValue:@(circleGeometry.radius) forKey:@"geometryRadius"];
         [dict setValue:[circleGeometry.center dictionaryValue] forKey:@"geometryCenter"];
-        [dict setValue:@"Circle" forKey:@"type"];
+        [dict setValue:@"circle" forKey:@"type"];
```

Comment:
we pass down the type as `circle` and `polygon` from the server and parse it as such, however, we were setting the dict with the capitalized version, this causes issues. We need to make sure this does not impact our cross-platform change.

Reply:
I vaguely remember fixing this (maybe in RN, changing it to expect capitalization)

Reply:
hmm it might be easier to just change the fromJson within the native SDK to accept both capitalized and non capitalized names of the shape to avoid introducing knock on effects downstream, this can help avoid introducing a breaking change.